samples/basic/blinky_pwm/src/main.c

```diff
@@ -53,16 +53,15 @@ void main(void)
 	 * Keep its value at least MIN_PERIOD_USEC * 4 to make sure
 	 * the sample changes frequency at least once.
 	 */
-	printk("Calibrating for device %s channel %d...\n",
-	       PWM_LABEL, PWM_CHANNEL);
+	printk("Calibrating for channel %d...\n", PWM_CHANNEL);
```

Comment:
Printing `dev->name` is still a useful thing to do, especially in a basic sample. I'd like to keep this. We're not going to get rid of names everywhere, right? I thought the idea was to be able to configure them out or make them the empty strings to save ROM, but in Kconfig.